src/check/runner/reporter/RunExecution.ts

```diff
@@ -96,14 +96,14 @@ export class RunExecution<Ts> {
       // encountered a property failure
       return {
         failed: true,
-        interrupted: this.interrupted,
+        interrupted: false,

```

Comment:
Those changes have to be double-checked. Ideally types should prevent us from such bugs 🤔